### PR TITLE
docs: v0.16 release

### PR DESCRIPTION
**Pull request by travisgroth**
## Summary

Update Changelog and upgrade notes

## Related issues

n/a

## Checklist

- [ ] reference any related issues
- [x] updated docs
- [ ] updated unit tests
- [ ] updated UPGRADING.md
- [x] add appropriate tag (`improvement` / `bug` / etc)
- [ ] ready for review

### CODE REVIEW

File: docs/docs/CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,508 @@
...
+- Docs: Add spdy annotation [\#2751](https://github.com/pomerium/pomerium/pull/2751) (@backport-actions-token[bot])
+- Docs: Add spdy annotation [\#2747](https://github.com/pomerium/pomerium/pull/2747) (@alexfornuto)
+- Docs: Ingress Controller [\#2745](https://github.com/pomerium/pomerium/pull/2745) (@backport-actions-token[bot])
+- Docs: Update Traefik Example Headers [\#2741](https://github.com/pomerium/pomerium/pull/2741) (@backport-actions-token[bot])
+- Docs: Update Community Page [\#2731](https://github.com/pomerium/pomerium/pull/2731) (@backport-actions-token[bot])
+- Minor fix in routes documentation [\#2721](https://github.com/pomerium/pomerium/pull/2721) (@backport-actions-token[bot])
+- Docs: Reference gRPC API Docs [\#2720](https://github.com/pomerium/pomerium/pull/2720) (@backport-actions-token[bot])
+- Update architecture.md [\#2707](https://github.com/pomerium/pomerium/pull/2707) (@backport-actions-token[bot])
+
+## [v0.15.6](https://github.com/pomerium/pomerium/tree/v0.15.6) (2021-11-04)
```
Comment (travisgroth) [inline docs/docs/CHANGELOG.md:137]: This entry needs to be manually consolidated with v0.16.0 when we're ready to release..  The misplaced tag confuses the changelog generator.

File: docs/docs/upgrading.md [hunk 2]
```diff
@@ -7,6 +7,29 @@ description: >-
...
+### Policy for Device Identity
+
+This release of Pomerium adds the ability to set policy based on system registration via [WebAuthN](https://www.w3.org/TR/webauthn-2/#registration-extension).
+
+See [device identity](/docs/topics/device-identity.md) for more details.
+
+### HTTP PPL Criteria
+
+`http_path` and `http_method` are now supported for matching HTTP requests in policies. See [PPL](/docs/topics/ppl.md#criteria) for more details.
+
```
Comment (desimone) [inline docs/docs/upgrading.md:21]: Empty block

**Review by desimone — APPROVED** [review]
LGTM sans an empty heading block

File: docs/docs/upgrading.md (hunk 2 above)
Comment (travisgroth) [reply docs/docs/upgrading.md:21]: 👀

File: docs/docs/upgrading.md [hunk 3]
```diff
@@ -7,6 +7,28 @@ description: >-
...
+## Breaking
+
+### Self Signed fallback certificates
+
+When selecting a TLS certificate for a listener, Pomerium attempts to locate one by iterating through the provide certs and searching for a SAN match against a service URL such as `databroker_service_url`.
```
Comment (alexfornuto) [inline docs/docs/upgrading.md:26]: 1. Would "listener" be analogous to the `from` value in this context?
2. I'm not following how this relates to `databroker_service_url`.

File: docs/docs/upgrading.md [hunk 4]
```diff
@@ -7,6 +7,28 @@ description: >-
 
 # Since 0.15.0
 
+## New
+
+### Policy for Device Identity
+
+This release of Pomerium adds the ability to set policy based on system registration via [WebAuthN](https://www.w3.org/TR/webauthn-2/#registration-extension).
```
Comment (alexfornuto) [inline docs/docs/upgrading.md:14]: @travisgroth I don't understand the reasoning to link to that specific header.

File: docs/docs/upgrading.md (hunk 3 above)
Comment (alexfornuto) [reply docs/docs/upgrading.md:26]: Discussed with @desimone, will suggest revised copy in a new PR.

File: docs/docs/upgrading.md (hunk 4 above)
Comment (desimone) [reply docs/docs/upgrading.md:14]: IMHO, we should link to the wikipedia entry. https://en.wikipedia.org/wiki/WebAuthn

**Pull request merged**